docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -62,13 +70,14 @@ If you are using the Pod Security admission controller, then MinIO and the tempo
 
 Using a custom namespace solves problems later on because you do not have to overwrite the default namespace.
 
-1. Create a unique Kubernetes namespace, for example `tempo-test`:
+1. Create a unique Kubernetes namespace, for example `tempo-test`, and switch your local context to use it:
 
 ```bash
 kubectl create namespace tempo-test
+kubens tempo-test
 ```
 
-   For more details, see the Kubernetes documentation about [Creating a new namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
+For more details, see the Kubernetes documentation about [Creating a new namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
```

Comment:
Restoring the spacing here. This spacing makes aligns this paragraph with the numbered step. 
```suggestion
  For more details, see the Kubernetes documentation about [Creating a new namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace).
```